src/VRPaths_openvr.cpp

```diff
@@ -0,0 +1,262 @@
+// NOTE: the following is an amalgamation of vrpathregistry_public.cpp, strtools_public.cpp, envvartools_public.cpp, and pathtools_public.cpp
+//       from openvr, to avoid the questions of "Can we already call this somehow?"
+//       or "How can we directly integrate this in our build system?"
+// TODO: Test on OSX/Linux, because our build system is probably not outputting the macros that this wants for them.
+//       I already had to change WIN32 to _WIN32 for the initial include of windows.h
```

Comment:
I believe this is technically a regression in cross-platform compatibility (which is somewhat important because i know of at least two efforts to provide an alternative to windows specific named pipes for other platforms), but I don't think it should be too difficult to either update the build system or find the correct defines to use with `#ifdef` when the time comes.